api/ocm/tools/signing/store.go

```diff
@@ -0,0 +1,211 @@
+package signing
+
+import (
+	"io"
+	"os"
+	"sync"
+
+	"github.com/mandelsoft/filepath/pkg/filepath"
+	"github.com/mandelsoft/goutils/errors"
+	"github.com/mandelsoft/goutils/sliceutils"
+	"github.com/mandelsoft/vfs/pkg/vfs"
+	"golang.org/x/exp/maps"
+
+	"ocm.software/ocm/api/ocm/compdesc"
+	metav1 "ocm.software/ocm/api/ocm/compdesc/meta/v1"
+	"ocm.software/ocm/api/utils"
+	common "ocm.software/ocm/api/utils/misc"
+	"ocm.software/ocm/api/utils/runtime"
+)
+
+type VerifiedStore interface {
```

Comment:
Please add some comments to these and explain what these new store things are.

Reply:
yes I'll add this tomorrow.